{
  "Header": {
    "Original": "CVE-2025-2487 (A flaw was found in the 389-ds-base LDAP Server. This issue occurs whe ...)",
    "ID": "CVE-2025-2487",
    "Description": "(A flaw was found in the 389-ds-base LDAP Server. This issue occurs whe ...)"
  },
  "Annotations": [
    {
      "Original": "- 389-ds-base 3.1.2+dfsg1-1 (bug #1100994)",
      "Type": "package",
      "Package": "389-ds-base",
      "Kind": "fixed",
      "Version": "3.1.2+dfsg1-1",
      "BugNo": 1100994
    },
    {
      "Original": "[bookworm] - 389-ds-base \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bookworm",
      "Package": "389-ds-base",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[bullseye] - 389-ds-base \u003cpostponed\u003e (need priviligied user; DoS)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "389-ds-base",
      "Kind": "postponed",
      "Description": "need priviligied user; DoS"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=2353071",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=2353071"
    }
  ]
}